client/text_extraction.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package notion
+package client
```

Comment:
Idea for another PR - the text extraction is currently probably not the fastest since we go through JSON. It would be nicer if we cast `notion.Block` to the corresponding object (e.g. `notion.ParagraphBlock` and others) and take out the value we want.

Reply:
Yup, I had that in mind actually. The main reason for JSON was the speed of implementation. Casting blocks into corresponding objects needs a lot of switches or figuring out the field with reflection made it a bit messy, given the number of block types.